input/_Layout.cshtml

```diff
@@ -14,6 +14,7 @@
             }
         }
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.26.0/themes/prism.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.22.0/themes/prism-tomorrow.min.css" />
```

Comment:
Is the Prism theme required? It affects syntax highlighting colors and button seems to also work without it.

Reply:
well no i will remove it i added it when i was following the doc 
without the css
before
![image](https://github.com/statiqdev/Docable/assets/28777412/04eccedd-61f0-4b47-b7dc-f54f5a4cdae7)

after
![image](https://github.com/statiqdev/Docable/assets/28777412/d397a276-8f1a-47a4-876c-b81a72e611e1)

let me remove it